photo-app/app/models/payment.rb

```diff
@@ -3,7 +3,7 @@ class Payment < ApplicationRecord
   belongs_to :user
 
   def self.choose_month
```

Comment:
don't understand this method name and what it does.
From the method name, it seems to be "give me the month". But, there is no input and it seems to be remapping the month data to index?

Reply:
`card_expiry_month` is way better than what you have now